src/coreclr/src/pal/src/libunwind/libunwind-version.txt

```diff
@@ -1,4 +1,5 @@
-v1.5-rc2
-https://github.com/libunwind/libunwind/releases/tag/v1.5-rc2
-+https://git.alpinelinux.org/aports/tree/main/libunwind/fix-aarch64-sigset_t.patch
-+https://github.com/libunwind/libunwind/pull/179
+v1.5-rc1-28-g9165d2a1
```

Comment:
Should it be `-rc2-` instead of `-rc1-`?

Reply:
You'd think ...
```bash
$ git describe origin/master --tags
v1.5-rc1-28-g9165d2a1

$ git log --format=oneline v1.5-rc2..master
9165d2a150d707d3037c2045f2cdc0fabd5fee98 (HEAD -> master, origin/master, origin/HEAD) Fix Windows abstraction SIZEOF_*
933464ed301932afd1b8ce3b97f473901058f859 Fix Travis job matrix
bac2b10878f729f9eaa9076f879bf40f1d58174b Add Windows build support
693f30a7c3c4092228de1430e005b435aa7656ce Add simple Windows platform abstraction layer
51bc8b160e7bd37879cb0b60a862d1640b860aed Add freebsd-elf* headers for WIndows compile
6a9e8725d52c66d17a2dcbf5be3685469934dbd2 Remove aux directory
5df1555828add4e256cb4bba49fffffc22127c34 aarch64: Fix __sigset build issue on muslC
6f2cc232b8d69546c6ec1df486125b47dc879b9f Define UNW_EMPTY_STRUCT for non-GNUC
df512cd4a7db78ae320f91611bef1689a219d8d2 Fix warnings
45fc222a00f239a16f1420c4446113e8a730b75a Use C11 atomics
da8dc856ab5646e04160060aae9425db3f5428ce Use C11 alignas(x)
d627f4aea024f7d1bfaa7ce7293379316c7713ea Use C11 _Thread_local
8321b7820096c44750cf5360654dcaae99827439 Only use one set of macros in Gexpr.c
29b86b7c50e2bc43c37578a681064d5292e975b8 Remove #include compiler.h in public header
a52e5e7cb7948e536eca6c29bfa0d56f57576575 Fix MSVC UNW_REMOTE_ONLY compilation errors
52c6488da816857de4c782bc5217defdb4c0bea3 Fix a memory leak issue
cb5016341a550849c126c97b9da96e674b396f70 src/arm/Gex_tables.c: resolve possible null pointer dereference
13f62c917e1d725162362dd6439becee2e5f3e08 Fix error in aarch64 unw_sigcontext
93ae875192ffd9bc5ef6853f04b1529759a2d4d4 Fix off-by-one error in x86_64 stack frames
5f83cc31c0bca58e1fab8eac036e6714ff8c2931 Reflect current status of Solaris x64 in the README
fd38b8a7ea74d2a38be659f439717e850979da20 Update README to reflect Solaris x86-64 status w/ this PR
7d335e17489c6d55f633ee9b8a8c9a0d3507627a Stop stepping on Solaris and illumos after reaching an IP of 0
cce9cc0f963a5897a41e6746dd336e4548ebf741 Fix Gtest-trace output for illumos/Solaris
8edf3a5e0cdc47a3229afd09016dd7c377bcf691 Fix signal stacks on illumos/Solaris
0190b9c29a6cb77fe80a30c7f0deea6181d499ea Add additional symbols to ignore on illumos and Solaris
91830952e35976d2c000e16cb971cb17b4481522 Treat PT_SUNW_UNWIND sections on illumos and Solaris as equivalent to PT_GNU_EH_FRAME
e5ecd9cf89de41c6e272d51c5e1f494439afe2b0 Update Gtest-bt for illumos and Solaris
11fd461095ea98f4b3e3a361f5a8a558519363fa coredump: Unmap elf image before mapping another (#169)

$ git log --format=oneline master..v1.5-rc2
fd35350d45b84c0032a9078220e8d222e45a1aaf (tag: v1.5-rc2) Fix off-by-one error in x86_64 stack frames
888cb924e24a54ae6dddf055404991d27ece68a0 Reflect current status of Solaris x64 in the README
3d055fefe34b972282e97ad6cd2da48b6dd9bc87 Update README to reflect Solaris x86-64 status w/ this PR
5e889a42faa694e94096a46033f69c19d67229c4 Stop stepping on Solaris and illumos after reaching an IP of 0
04cd751deae301c606b382dba9e9207bdefc7a9d Treat PT_SUNW_UNWIND sections on illumos and Solaris as equivalent to PT_GNU_EH_FRAME
6bba070f49138f0569f29088207322fde03e7b04 Fix Gtest-trace output for illumos/Solaris
d433df57a7fa72cda1513a77683e932e25845e79 Fix signal stacks on illumos/Solaris
af5bef639ea471f9224fec71a53d276286a02085 Add additional symbols to ignore on illumos and Solaris
7783f6379b9d59d07071d725aa14181df574236a Update Gtest-bt for illumos and Solaris
a8b94341faf8b04d9041bbaecb54481b70295a6c coredump: Unmap elf image before mapping another (#169)
d866775860bcc8d26406c4b73bd1e285c7ea0c23 Update README version
```

So master is missing d866775860bcc8d26406c4b73bd1e285c7ea0c23 , and the order of commits is slightly different.